src/stores/index.js

```diff
@@ -1020,6 +1031,76 @@ class Store {
     })
   }
 
+  getLeaderBoard = (_payload) => {
+    
+    const account = store.getStore('account')
+    const web3 = new Web3(store.getStore('web3context').library.provider);
+    const memes = this.getMemes();
+
+    async.map(memes, (meme, callback) => {
+        this._getLeaderBoardData(web3, meme, callback);
+    }, async(err, leaderboardData) => {
+      if(err) {
+        return emitter.emit(ERROR, err);
+      }
+      console.log(leaderboardData);
+      leaderboardData.sort(function(a, b) {
+        return a.score - b.score; 
+      })
+
+      leaderboardData.reverse();
```

Comment:
why don't you simply reverse the sort function and remove array reverse?

Reply:
Okay, I will do that. I told you I am not a JS veteran :D

Reply:
FIXED